File: client-sdk/spicepy-sdk-sample/main_cloud.py
# Install with: pip install git+https://github.com/spiceai/spicepy
from spicepy import Client

client = Client(
    api_key='API_KEY',
    flight_url="grpc+tls://flight.spiceai.io"
)
data = client.query('show tables;', timeout=5*60)
pd = data.read_pandas()

print(pd)

File: client-sdk/spice-rs-sdk-sample/src/main_cloud.rs
use spiceai::{ClientBuilder, StreamExt};

#[tokio::main]
async fn main() {
    let mut client = ClientBuilder::new()
        .api_key("API_KEY")
        .flight_url("https://flight.spiceai.io")
        .build()
        .await
        .unwrap();

    let mut flight_data_stream = client
        .query("show tables;")
        .await
        .expect("Error executing query");

    while let Some(batch) = flight_data_stream.next().await {
        match batch {
            Ok(batch) => {
                /* process batch */
                println!("{:?}", batch)
            }
            Err(_) => { /* handle error */ }
        };
    }
}
